portfolio/src/main/webapp/js/script.js

```diff
@@ -92,16 +92,21 @@ function addRandomHobby() {
   hobbyContainer.innerText = hobby;
 }
 
-// Intializing the carousel
+// Intializing materialize components
 document.addEventListener('DOMContentLoaded', function() {
-    var elem = document.querySelector('.carousel');
-    M.Carousel.init(elem, {
+    var element = document.querySelector('.carousel');
```

Comment:
Use let instead of var.
As per the google js style guide (https://google.github.io/styleguide/jsguide.html#features-use-const-and-let) use const wherever possible; use let if you need to reassign the variable later; never use var.